celery/backends/database/extensions.py

```diff
@@ -0,0 +1,23 @@
+"""Schema extensions for Celery result backend tables."""
+
+from abc import ABC, abstractmethod
+
+from sqlalchemy import JSON, MetaData, Table
+
+
+class SchemaExtension(ABC):
```

Comment:
that is interesting. in which version it was introduced?

Reply:
Oh, if you meant ABC, that showed up in Python 3.4 as the easier way to define abstract base classes.